File: Alg05.py
# Escrever um algoritmo que leia o nome de um aluno e as notas
# das três provas que ele obteve no semestre. No final informar o nome do
# aluno e a sua média (aritmética).

# Data: 17/02/2020
# Autor: Gustavo Alves da Silva

nome = input('Insira o nome do aluno: ')

notas = []

for i in range(3):
   notas.append(float(input('Digite o valor da nota: '.format(i))))

soma_notas = 0.0

for i in range(3):
    soma_notas = soma_notas + notas[i]

media = soma_notas / 3

print('Aluno: {}'.format(nome))
print('Média: {:.1f}'.format(media))

File: Alg05.java
/*
 * Escrever um algoritmo que leia o nome de um aluno e as notas
 * das três provas que ele obteve no semestre. No final informar o nome do
 * aluno e a sua média (aritmética).
 */
package main;

import java.util.Scanner;

/**
 *
 * @author Gustavo Alves da Silva 
 * @date 20/08/2013
 */
public class Alg05 {
    public static void main(String[] args) {
        double nota1, nota2, nota3, media;
        String nome;
        
        Scanner ent = new Scanner(System.in);
        
        System.out.println("Insira o nome do aluno: ");
        nome = ent.nextLine();
        
        System.out.println("Nota da Prova 1: ");
        nota1 = ent.nextDouble();
        
        System.out.println("Nota da Prova 2: ");
        nota2 = ent.nextDouble();
        
        System.out.println("Nota da Prova 3: ");
        nota3 = ent.nextDouble();
        
        media = (nota1 + nota2 + nota3) / 3;
        
        System.out.printf("Aluno: %s", nome);
        System.out.printf("\nMédia: %.1f\n", media);
    }
            
}
